ANN_MNIST/data.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import pandas as pd
+
+
+def get_data():
+    X = pd.read_csv("data/imageMNIST.csv", decimal=",", header=None)
+    X = np.array(X)
+    #X, X.shape
+
+    y_raw = pd.read_csv("data/labelMNIST.csv", decimal=",", header=None)
+    y_raw = np.array(y_raw)
+    #y_raw, y_raw.shape
```

Comment:
Pode retirar essa linhas. Usie elas em main para verificar on the run como estavam ficando os dados.